src/components/Breadcrumb/__test__/breadcrumb.spec.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Breadcrumb from './../index';
+
+describe('<Breadcrumb/>', () => {
+    it('should fallback to href to href="javascript:void(0);" if the prop is not provided', () => {
```

Comment:
should fallback the href prop to href="javascript:void(0);" if the prop is not provided